Shoot/Shoot.xcodeproj/project.pbxproj

```diff
@@ -462,12 +463,12 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				ASSETCATALOG_COMPILER_LAUNCHIMAGE_NAME = LaunchImage;
 				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = GHPBX39444;
 				INFOPLIST_FILE = Shoot/Info.plist;
 				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
 				ONLY_ACTIVE_ARCH = YES;
-				PRODUCT_BUNDLE_IDENTIFIER = org.aerogear.shoot;
+				PRODUCT_BUNDLE_IDENTIFIER = org.aerogear.cookbook.shoot;
```

Comment:
is this intended or only because you had problems with the previous one?

Reply:
Both, I would like to have all cookbook using cookbook in the bundle and also because we had the previous one ;)

Reply:
If we want to use this new identifier, the google project used to share the photos has to be configured to use a new redirect url. Not sure who has the credentials.
Also, Facebook app and keycloack server will need to be updated too as they both use current bundle identifier in configurations.